backend/src/__test__ /utils/helpers.js

```diff
@@ -5,12 +5,10 @@ export const loginUser = async (
   email = 'sarah@email.com',
```

Comment:
Uh oh! I almost thought you have secrets in your code base. I tried the credentials and got:
```
"message": "User not found",
```
:relieved: Whew!